ckanext/validation/jobs.py

```diff
@@ -17,6 +17,11 @@
 
 from ckanext.validation.model import Validation
 
+try:
+    from ckanext.canada.tabulate import CanadaCSVParser as CSVParser
+except ImportError:
+    from tabulator.parsers.csv import CSVParser
```

Comment:
Instead of conditionally importing something from our theme extension (and silently ignoring failures) it would be much better to offer an interface that our extension could call to set up these custom options.